lib/utils/generateFinalBridgeObject.ts

```diff
@@ -0,0 +1,42 @@
+import type { ISbStoriesParams, StoryblokBridgeConfigV2 } from "@storyblok/js";
+
+export interface RawCodeItem {
+  options?: {
+    apiOptions?: ISbStoriesParams;
+    bridgeOptions?: StoryblokBridgeConfigV2;
+  };
+}
+export function generateFinalBridgeObject(
+  rawCode: RawCodeItem[]
+): StoryblokBridgeConfigV2 {
+  let mergedOptions = {
+    resolveRelations: [],
+  };
+
+  function addToResolveRelations(
+    resolveRelations: string[] | string | undefined
```

Comment:
I'm personally not a fan of explicitly stating that a function param could be `undefined`, as opposed to marking it with the `?`, but this is a matter of preferences 😊 If this sticks with the current coding guidelines, its 👌